storage/doc.go

```diff
@@ -246,12 +240,52 @@ as the documentation of GenerateSignedPostPolicyV4.
 
 Errors
 
-Errors returned by this client are often of the type [`googleapi.Error`](https://godoc.org/google.golang.org/api/googleapi#Error).
-These errors can be introspected for more information by using `errors.As` with the richer `googleapi.Error` type. For example:
+Errors returned by this client are often of the type googleapi.Error.
+These errors can be introspected for more information by using errors.As
+with the richer googleapi.Error type. For example:
 
 	var e *googleapi.Error
 	if ok := errors.As(err, &e); ok {
 		  if e.Code == 409 { ... }
 	}
+
+See https://pkg.go.dev/google.golang.org/api/googleapi#Error for more information.
+
+Retrying failed requests
+
+Methods of this package may use exponential backoff to retry calls
+that fail with transient errors. Retrying continues indefinitely unless the
+controlling context is canceled, the client is closed, or a non-transient error
+is received. See context.WithTimeout and context.WithCancel.
+
+Retry strategy in this library follows best practices for Cloud Storage. By
+default, only idempotent operations are retried, exponential backoff with jitter
+is employed, and only transient network errors and response codes defined as
+transient by the service and will be retried. See
+https://cloud.google.com/storage/docs/retry-strategy for more information.
+
+Users can configure non-default retry behavior for a particular operation (using
```

Comment:
"You can customize the retry behavior for particular operations by using `BucketHandle.Retryer` and `ObjectHandle.Retryer`. You can also customize the retry behavior for all calls made by a client by using `Client.SetRetry`."

Also, what's meant by "particular operations"? Is this talking about the operations that are idempotent?

Reply:
"For particular operations" meaning "for a single library call". Maybe the latter would be clearer.

Reply:
Can we replace "particular operations" with "single library calls"